programs/mango-v4/src/instructions/account_buyback_fees_with_mngo.rs

```diff
@@ -94,6 +96,14 @@ pub fn account_buyback_fees_with_mngo(
         now_ts,
         mngo_oracle_price,
     )?;
+    emit!(TokenBalanceLog {
+        mango_group: ctx.accounts.group.key(),
+        mango_account: ctx.accounts.account.key(),
+        token_index: mngo_bank.token_index,
+        indexed_position: account_mngo_token_position.indexed_position.to_bits(),
+        deposit_index: mngo_bank.deposit_index.to_bits(),
+        borrow_index: mngo_bank.borrow_index.to_bits(),
+    });
```

Comment:
Technically there are four token position changes in this instruction. The important ones are this one but also the user's USDC position change in line 134 - could you add the log to that one too?

Reply:
Thanks Christian, I missed that.